File: Programming-languages-and-laboratory/Python/Exercises/18_compito_java_20170911/Individuo.py
from Iscritto import Iscritto


class Individuo(Iscritto):
    def __init__(self, tipo, codice, nome_cognome, indirizzo, nome_account, password):
        super().__init__(tipo, codice, indirizzo, nome_account, password)
        self._nome_cognome = nome_cognome

    def get_nome(self):
        return self._nome_cognome

    def __str__(self):
        return str(self._codice) + "\t" + self._nome_cognome + "\t-\t-\t" + self._indirizzo + "\t" + self._nome_account + "\t" + str(len(self._password))

File: II - Linguaggi di programmazione e laboratorio/Esame/Java/18_compito_java_2017_09_11/Individuo.java
public class Individuo extends Iscritto {
	private String nomeCognome;

	public Individuo(String tipo, int codice, String nomeCognome, String indirizzo, String nomeAccount,
			String password) {
		super(tipo, codice, indirizzo, nomeAccount, password);
		this.nomeCognome = nomeCognome;
	}

	public String getNome() {
		return nomeCognome;
	}

	public String toString() {
		return codice + "\t" + nomeCognome + "\t-\t-\t" + indirizzo + "\t" + nomeAccount + "\t" + password.length();
	}

	public String toStringFollowerNumber() {
		return codice + "\t" + nomeCognome + "\t" + seguaci.size();
	}
}